robotbase/src/main/java/frc/robot/controls/DriverControls.java

```diff
@@ -81,7 +82,9 @@ public void mapControls() {
         new ConditionalCommand(
             new AmpScore(), new AmpPrepose(), () -> Robot.state.isInState(State.AMP_PRE_POSE)));
 
-    m_rightTriggerShoot.whileTrue(new DefaultPivot().andThen(new PivotStop()));
+    m_rightTriggerPrime.whileTrue(new ShooterSetRPMs(5000, 5000));
+    m_rightTriggerShoot.whileTrue(
```

Comment:
I would let the shooters and intake run until we let go in case the note is in a bad position